### ISSUE TITLE
Fix limitation regarding aria-expanded and aria-details

### ISSUE THREAD

**Issue by lukewarlow** [keithamus/invoker-buttons-proposal#76]
As mentioned under limitations in the readme the polyfill doesn't handle the aria attributes. Ideally it should actually handle those, I think at least aria-expanded should be relatively simple to polyfill.
